coalib/output/ConsoleInteraction.py

```diff
@@ -505,7 +505,7 @@ def acquire_settings(log_printer, settings_names_dict):
 
     result = {}
     for setting_name, arr in sorted(settings_names_dict.items(),
-                                    key=lambda x: join_names(x[1][1:])):
+                                    key=lambda x: (join_names(x[1][1:]), x[0])):
```

Comment:
I think this won't change order as the comparison for `join_names(x[1][1:])` still has priority^^ (though I'm not sure what's inside `arr`...)